build.gradle

```diff
@@ -917,6 +919,45 @@ project('tcp-client-server') {
 	}
 }
 
+project('testcontainers-rabbitmq') {
+	description = 'Testcontainers RabbitMQ Sample'
+
+	apply plugin: 'org.springframework.boot'
+
+	bootRun {
+		main = 'org.springframework.integration.samples.testcontainersrabbitmq.TestcontainersRabbitmqApplication'
+	}
+
+	dependencies {
+		compile "org.springframework.boot:spring-boot-starter-integration"
+		compile "org.springframework.integration:spring-integration-amqp"
+		compile "org.springframework.boot:spring-boot-starter-json"
+
+		testCompile('org.springframework.boot:spring-boot-starter-test') {
+			exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+		}
+		testCompile 'org.springframework.integration:spring-integration-test'
+		testCompile 'org.springframework.integration:spring-integration-test-support'
```

Comment:
I don't think we need this one.
It has to come transitively from the `spring-integration-test`.

Reply:
removed